packages/metastream-app/src/locale/ja.ts

```diff
@@ -92,6 +93,11 @@ export default {
   once: '一度',
   openInBrowser: 'ブラウザで開く',
   openInPopup: 'ポップアップで表示する',	
+  patreonLogin: 'Patronでログイン',	
```

Comment:
I started learning a very small amount of Japanese in the past year. Why would katakana not be used in this case for "Patron"? I trust your translations, I'm just interested to learn. 🙂